### DISCUSSION TITLE
Has anyone had issues with an "invisible pointer" with Barrier v2.3.3 on Windows 10 and Ubuntu 21.04?

### DISCUSSION THREAD

**Discussion by sd-newbe** [debauchee/barrier#1314] [Q&A]
Help! Anybody
Installed Barrier 2.3.3 on Win10 and Ubuntu 21.04
It works but I can't see the pointer on Ubuntu screen. Checked the log on Ubuntu and it says "entering screen" and "leaving screen"
But invisible pointer!!!! Has anybody experienced this problem?
Solution?
Thanks
Sam

**Comment by shymega** [comment]
Not an answer, but I've adjusted your question's title to be a *little* more descriptive. I think in the future, you may want to think about titles for issues/questions in terms of wording to gain the attention needed to solve your issue :smile:

**Reply by sd-newbe** [reply]
Thank you. I am new in LINUX and Github and I do appreciate all help.
Hope somebody had experienced the problem I have and can solve my strange.
I may have add that this problem occurs only in my UBUNTU 21.04, not in windows.
Thanks
Sam

**Comment by medinasp** [comment]
I´ve passing by the same problem, any solved this?

**Reply by shymega** [reply]
Barrier doesn't seem maintained anymore, but users may have found a solution.

**Reply by medinasp** [reply]
Is there any other place where I can found this solution?

**Reply by shymega** [reply]
I don't know. The active maintainers (including me) forked Barrier some time back, if anyone on that repo has found a solution - I do not know.

Bear in mind v2.3.3 is vulnerable, and you should be on v2.4.0, or v.2.3.4 at the very least.

**Reply by medinasp** [reply]
Thank u very much, my friend.
